src/main/java/org/folio/circulation/rules/ItemLocation.java

```diff
@@ -1,17 +1,17 @@
 package org.folio.circulation.rules;
 
 /**
- * Store the UUID of the shelving location.
+ * Store the UUID of the location.
  */
-public class ShelvingLocation {
-  /** UUID of the shelving location. */
+public class ItemLocation {
```

Comment:
@julianladisch I want to provide you some information about this renameing "Location.java ---> ItemLocation.java":
in some places we have usage of two Location objects (Location from '**org.folio.circulation.rules**' and from '**org.folio.circulation.domain**').
So, for not renaming domain object, which used everywhere in the module, I renamed this "Location.java" from **org.folio.circulation.rules**'  to "ItemLocation.java", because there is only one usage fo this class.

Reply:
Yes, this is better than using the fully qualified name to resolve the name ambiguity. https://docs.oracle.com/javase/tutorial/java/package/usepkgs.html